src/world.rs

```diff
@@ -152,6 +166,8 @@ impl World {
     /// reserved entities to become real entities before proceeding. This can also be done
     /// explicitly by calling `flush`.
     ///
+    /// This is equivalent to `reserve_entities(1)`, but more efficient.
+    ///
     /// Useful for reserving an ID that will later have components attached to it with `insert`.
```

Comment:
Rather than duplicating the comment (which will certainly lead to divergence), let's direct the reader to `reserve_entities`, as is done in e.g. `insert_one` vs. `insert`.